src/page-multi-facility/ReferenceDataModal/index.tsx

```diff
@@ -76,14 +76,6 @@ const ReferenceDataModal: React.FC<Props> = ({
   } = useFacilities();
 
   async function handleClose() {
-    rejectionToast(
```

Comment:
Why did we remove this toast?

Reply:
it's been moved up to the `onClose` function instead, to keep things together and make it more reusable